### PR TITLE
Language rework

### PR DESCRIPTION
**Pull request by comradekingu**
Norwegian Bokmål added as language (translation is done)
A lot of the changes that seem interchangeable are intended to make for fewer errors in translation.

### CODE REVIEW

File: muninforAndroid/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -175,22 +177,22 @@
...
-	<string name="text53">Update by Wifi only</string>
+	<string name="text53">Only update on Wi-Fi</string>
 	<string name="text54">Estimated data consumption: ?? /day</string>
 	<string name="text55">Refresh rate</string>
 	<string name="text56">Select nodes to watch</string>
 	<string name="text57">10 minutes/30 minutes/1 hour/2 hours/5 hours/10 hours/24 hours</string>
 	<string name="text58"> critical / criticals /&amp; / warning / warnings /on / node/ nodes</string>
 	<string name="text60">Continue</string>
 	<string name="text61">Add</string>
-	<string name="text62">There\'s no label yet</string>
```
Comment (quentin-st) [inline muninforAndroid/src/main/res/values/strings.xml:171]: This message is displayed on the labels screen where no label has been created yet - "Unlabeled" seems irrelevant, isn't it?

File: muninforAndroid/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -222,30 +224,30 @@
 	<string name="move_down">Move down</string>
 	<string name="move_left">Move left</string>
 	<string name="rate">Rate the app</string>
-	<string name="rate_long">If you like the app, would you mind rating it on Google Play? Thank you!</string>
+	<string name="rate_long">If you like the app, rate it on Google Play. Thank you.</string>
```
Comment (quentin-st) [inline muninforAndroid/src/main/res/values/strings.xml]: This new message looks way too formal and serious, don't you like the older one?

File: muninforAndroid/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -269,15 +271,14 @@
 	<string name="disableHDGraphs">Disable HD Graphs</string>
 	<string name="downloadError">Download error</string>
 	<string name="certificate_error">Your connection is not private</string>
-	<string name="alert_opensource">Did you know? Munin for Android is now open-source!</string>
+	<string name="alert_opensource">Munin for Android is now GPLv2 copyleft libre software.</string>
```
Comment (quentin-st) [inline muninforAndroid/src/main/res/values/strings.xml]: The previous sentence sounds better, don't you think?

File: muninforAndroid/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -300,9 +301,9 @@
 	<string name="notifications_device_code">Device code</string>
 	<string name="notifications_instructions">You have to configure each master so they can send notifications to this device.</string>
 	<string name="notifications_sendByMail">Send me the code and instructions by mail</string>
-	<string name="notifications_mailSent">Mail sent! Please check your inbox for it</string>
+	<string name="notifications_mailSent">E-mail sent. PCheck your inbox</string>
```
Comment (quentin-st) [inline muninforAndroid/src/main/res/values/strings.xml]: `PCheck your`: typo?

**Review by quentin-st — CHANGES_REQUESTED** [review]
Seems great, thank you for your contribution!

IMHO, some sentences were better before (some seem too formal now), but really great enhancements in general!

File: muninforAndroid/src/main/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [reply muninforAndroid/src/main/res/values/strings.xml]: It sounds like begging. From a neutral and considering the psychological aspect:
"Your enjoyment of this app is needed in rating this app, because it could help others in forming their decisions." Though it is much longer, I think it would lead to more reviews.

File: muninforAndroid/src/main/res/values/strings.xml (hunk 3 above)
Comment (comradekingu) [reply muninforAndroid/src/main/res/values/strings.xml]: It seems like overacting to me.

File: muninforAndroid/src/main/res/values/strings.xml (hunk 4 above)
Comment (comradekingu) [reply muninforAndroid/src/main/res/values/strings.xml]: Nice catch

File: muninforAndroid/src/main/res/values/strings.xml (hunk 2 above)
Comment (quentin-st) [reply muninforAndroid/src/main/res/values/strings.xml]: I really like this one!
I didn't quite like "If you like the app, rate it on Google Play. Thank you." because it looks like an imperative & too much informal statement (the two dots makes it weird IMHO)

File: muninforAndroid/src/main/res/values/strings.xml (hunk 3 above)
Comment (quentin-st) [reply muninforAndroid/src/main/res/values/strings.xml]: +1.
We may remove alert soon since it's been in place for a bunch of time

File: muninforAndroid/src/main/res/values/strings.xml (hunk 1 above)
Comment (comradekingu) [reply muninforAndroid/src/main/res/values/strings.xml:171]: No more so than "There\'s no label yet". However it could be neither (?)

**Review by quentin-st — APPROVED** [review]

**Comment by quentin-st** [comment]
Thank you for your contribution!
